src/coreclr/src/jit/optcse.cpp

```diff
@@ -1664,7 +1664,7 @@ class CSE_Heuristic
                 largeFrame = true;
                 break; // early out,  we don't need to keep increasing frameSize
             }
-#elif _TARGET_ARM32
+#elif TARGET_ARM
```

Comment:
Note: Functional change.

Reply:
@briansull - could you comment on this one?